AuthenticatorBridgeKit/AuthenticatorBridgeItemService.swift

```diff
@@ -124,4 +144,18 @@ public class DefaultAuthenticatorBridgeItemService: AuthenticatorBridgeItemServi
             insertRequest: insertRequest
         )
     }
+
+    public func sharedItemsPublisher() async throws ->
+        AnyPublisher<[AuthenticatorBridgeItemDataView], any Error> {
+        let fetchRequest = AuthenticatorBridgeItemData.fetchRequest()
+        fetchRequest.sortDescriptors = [NSSortDescriptor(keyPath: \AuthenticatorBridgeItemData.userId, ascending: true)]
+        return FetchedResultsPublisher(
+            context: dataStore.persistentContainer.viewContext,
+            request: fetchRequest
+        )
+        .asyncTryMap { items in
+            try await self.cryptoService.decryptAuthenticatorItemDatas(items)
+        }
+        .eraseToAnyPublisher()
```

Comment:
🤔 I think this can be improved to use `decryptAuthenticatorItemModels` by chaining the model mapping:
```suggestion
        .tryMap { dataItems in 
            return dataItems.compactMap { data in 
                guard let model = itemData.model else { 
                    return nil 
                }
                return model
            }
        }
        .asyncTryMap { itemModel in
            try await self.cryptoService.decryptAuthenticatorItemModels(items)
        }
        .eraseToAnyPublisher()
```

Reply:
👍 Great suggestion. I like this a lot better! I had been playing around with different ways of doing this, but for some reason didn't land on this one. Reduces duplication and cleans up the CryptoService's responsibilities 💯 

Adopted this in the latest.